Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -214,7 +214,7 @@
 			    }
 				catch [System.TimeoutException]
 				{
-					# Catch operation timeout and continue with force remove replica.
+					$app | Remove-ServiceFabricApplication -ForceRemove
```

Comment:
I think you still need to add -Force to bypass a confirmation prompt: [docs](https://docs.microsoft.com/en-us/powershell/module/servicefabric/remove-servicefabricservice?view=azureservicefabricps#optional-parameters)

Reply:
Correct. Missed while committing. Modified and tested the statement now.